src/content/docs/en/technology/overview/scroll-upgrades/feynman-upgrade.mdx

```diff
@@ -0,0 +1,75 @@
+---
+section: technology
+date: Last Modified
+title: "Feynman Upgrade"
+lang: "en"
+permalink: "technology/overview/scroll-upgrades/feynman-upgrade"
+whatsnext: { "Euclid Upgrade": "/en/technology/overview/scroll-upgrades/euclid-upgrade" }
+---
+
+### Overview
+This upgrade contains changes such as:
+- Improve compatibility with EVM:
+    - opcode `blockhash`
+    - pre-compiles `ecPairing`
+- Implemented:
```

Comment:
EIP-2935 and EIP-7623 should be also under "Improve compatibility with EVM".

Actually you can directly copy [the outline in the governance doc](https://forum.scroll.io/t/proposal-feynman-upgrade/957#p-2341-appendix-a-detailed-list-of-features-11).